nautobot/dcim/tests/test_api.py

```diff
@@ -3377,6 +3377,7 @@ class VirtualDeviceContextTestCase(APIViewTestCases.APIViewTestCase):
     def setUpTestData(cls):
         devices = Device.objects.all()
         vdc_status = Status.objects.get_for_model(VirtualDeviceContext)[0]
+        vdc_role = Role.objects.get_for_model(VirtualDeviceContext)[0]
```

Comment:
Latest test failure *might* be due to not finding a valid Role object? Hard to say for sure.